### PR TITLE
Update RestCallTests.java

### PR DESCRIPTION
**Pull request by idan-marciano-jfrog**
Fix testDeleteBuild test

### CODE REVIEW

**Comment by idan-marciano-jfrog** [comment]
I have read the CLA Document and I hereby sign the CLA

File: services/src/test/java/org/jfrog/artifactory/client/RestCallTests.java [hunk 1]
```diff
@@ -207,6 +208,13 @@ private String deleteBuild(String name) throws Exception {
                 .addQueryParam("artifacts", "1");
         return artifactory.restCall(deleteBuild).getRawBody();
     }
+    private String getBuilds() throws Exception {
```
Comment (yahavi) [inline services/src/test/java/org/jfrog/artifactory/client/RestCallTests.java:211]:
```suggestion
    }
    
    private String getBuilds() throws Exception {
```

**Review by yahavi — CHANGES_REQUESTED** [review]
Thanks, @idan-marciano-jfrog, looks good!
Please add an empty line between deleteBuild and getBuilds.

File: services/src/test/java/org/jfrog/artifactory/client/RestCallTests.java (hunk 1 above)
Comment (yahavi) [inline services/src/test/java/org/jfrog/artifactory/client/RestCallTests.java:211]:
```suggestion

    private String getBuilds() throws Exception {
```

**Pull request merged**